MauriceK1/flask-project
# Add tests

tests.test_db
- Add tests for database by creating timeline posts and asserting that they are correct

tests.test_app
- Add tests for home page
- Add tests relating to the /api/timeline_post GET and POST apis
- Add tests relating to the timeline page
- Add tests for malformed timeline post for missing name, empty content, and malformed email

mutation-test.sh
- Optional mutation testing for test_db and test_app
- Note that tests.test_db will create posts that show on the timeline page and need to be deleted manually if desired
